app/jobs/send_vaccination_center_confirmed_email_job.rb

```diff
@@ -0,0 +1,11 @@
+class SendVaccinationCenterConfirmationEmailJob < ApplicationJob
+  queue_as :default
```

Comment:
En terme de délai d'envoi du mail, il est important que ça soit casi instante ou qu'il y est un délai de 2 3 minutes je pense c'est pas très grave.
```suggestion
  queue_as :mailers
```